extensions/objects/templates/translatable-set-of-unicode-string-editor.component.ts

```diff
@@ -56,8 +56,8 @@ export class TranslatableSetOfUnicodeStringEditorComponent {
 
   getSchema(): {
     type: string;
-    items: { type: string; };
-    validators: { id: string; }[];
+    items: { type: string };
+    validators: { id: string }[];
     } {
```

Comment:
```suggestion
  } {
```

Reply:
I have created an interface in the file `translatable-set-of-normalized-string-editor.component.ts`, imported it into this file and used it, since both the types are identical.